<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail</class>
 <widget class="QMainWindow" name="Detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1001</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion services</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>281</height>
      </size>
     </property>
     <property name="title">
      <string>Rechercher services</string>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PbSearch">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radionom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radiotype">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>481</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Crit√®re de trie</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date superieur a:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioAlphaCroissant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordre alphabetique croissant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioAlphaDecroissant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordre alphabetique decroissant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date inferieur a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date egale a:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateSup">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEgale">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateInf">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix croissant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix decroissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Defaut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ordre alphabetique croissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ordre alphabetique decroissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tarifs croissant </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tarifs decroissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>471</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Imprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="mailling">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>600</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(167, 0, 0);</string>
     </property>
     <property name="text">
      <string>Reseaux sociaux</string>
     </property>
    </widget>
    <widget class="QLabel" name="image4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PbReturn">
    <property name="geometry">
     <rect>
      <x>742</x>
      <y>0</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>retour au menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
